src/Model/User.php

```diff
@@ -573,4 +573,18 @@ public function setRealRoles(array $roles)
 
         return $this;
     }
+
+    /**
+     * Returns the gender list.
+     *
+     * @return array
+     */
+    public static function getGenderList()
```

Comment:
What if someone already added  a method `getGenderList` and it is not static? Looks like a BC break

Reply:
Indeed I actually did that on one of my older projects.

Reply:
> What if someone already added  a method `getGenderList` and it is not static? Looks like a BC break

In this case, every new method is a BC break xD

Reply:
So should I change back the targeting branch to 4. ?

Reply:
yes

Reply:
I did the change.